public/index.html

```diff
@@ -5,17 +5,69 @@
     <link rel="icon" href="%PUBLIC_URL%/favicon.png" />
     <meta
       name="viewport"
-      content="width=device-width, initial-scale=1, viewport-fit=cover"
+      content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=0, viewport-fit=cover"
```

Comment:
Can you expand on what this change does?

Reply:
Also, user-scalable is a boolean:

![image](https://user-images.githubusercontent.com/4369024/137255094-f84202cd-b466-470f-ad8d-8a75dd1403be.png)

Reply:
Great, this means we can change some input font sizes back from 16px too right?